plugins/available/wttr.plugin.bash

```diff
@@ -0,0 +1,39 @@
+function wttr() {
```

Comment:
You're also missing the function header which describes how to use the function - please add it.